libmamba/include/mamba/core/solver.hpp

```diff
@@ -101,7 +114,7 @@ namespace mamba
         // Order of m_pool and m_solver is critical since m_pool must outlive m_solver.
         MPool m_pool;
         std::unique_ptr<::Solver, void (*)(::Solver*)> m_solver;
-        Queue m_jobs;
+        std::unique_ptr<solv::ObjQueue> m_jobs;
```

Comment:
Is there a reason why this should be a unique_ptr now?

Reply:
Yes, we want to isolate `libsolv` from the public API so we cannot know the `solv::ObjQueue` type.

Reply:
Yes, we want to isolate `libsolv` from the public API so we cannot know the `solv::ObjQueue` type.